server/embed/serve.go

```diff
@@ -224,6 +224,7 @@ func configureHttpServer(srv *http.Server, cfg config.ServerConfig) error {
 	// todo (ahrtr): should we support configuring other parameters in the future as well?
 	return http2.ConfigureServer(srv, &http2.Server{
 		MaxConcurrentStreams: cfg.MaxConcurrentStreams,
+		NewWriteScheduler:    http2.NewRandomWriteScheduler,
```

Comment:
Please add comment above the flag, describing why it's critical for any reasonable performance out of etcd grpc APIs and pointing on the issue.